packages/ui/package.json

```diff
@@ -48,7 +48,6 @@
     "@emotion/react": "^11.8.2",
     "@emotion/styled": "^11.8.1",
     "@mui/material": "^5.5.1",
-    "react": "^16.0.0",
```

Comment:
This should be here.

Reply:
Just use React `17.0.2`. If there is a package complaining, let it complain for now & we’ll deal with it later.